datadog_lambda/__init__.py

```diff
@@ -1,12 +1,4 @@
 from datadog_lambda.cold_start import initialize_cold_start_tracing
-import os
-
-
-if os.environ.get("DD_INSTRUMENTATION_TELEMETRY_ENABLED") is None:
-    # Telemetry is required for Appsec Software Composition Analysis
-    os.environ["DD_INSTRUMENTATION_TELEMETRY_ENABLED"] = os.environ.get(
-        "DD_APPSEC_ENABLED", "false"
-    )
```

Comment:
The reason this code was added here was we must ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` is set _before_ any ddtrace code is imported. Can we ensure that we the change you're proposing? How can we enforce it?